purchase_allowed_product/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# © 2016 Chafique DELLI @ Akretion
+# © 2017 Today Mourad EL HADJ MIMOUNE @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Account Invoice Allowed Product',
+    'summary': 'This module allows to select only products that can be '
+               'supplied by the supplier',
+    'version': '11.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'website': 'https://akretion.com',
```

Comment:
@hieulucky111 Use URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)